src/modules/endpoint/FeedbackEndpointBuilder.js

```diff
@@ -0,0 +1,64 @@
+// @flow
```

Comment:
I think we do not need a Builder for for one specific endpoint which is used only once.